test.sh

```diff
@@ -78,8 +76,7 @@ $RUN $PYTHON setup.py install
 
 # Install packages for tests
 $RUN $PIP install -r tests/requirements.txt
+if [[ $OS != "fedora" ]]; then $RUN $PIP install -U setuptools; fi
```

Comment:
Why this line?

Reply:
CentOS has some ancient setuptools installed, which [breaks](https://stackoverflow.com/questions/26589990/py-test-error-unrecognized-arguments-cov-ner-brands-cov-report-term-missi/41066450#41066450) `pytest` and `pytest-cov` integration.

Reply:
Let's add a comment then, please.

Reply:
Right, done